src/Microsoft.DotNet.Wpf/cycle-breakers/PresentationFramework/PresentationFramework.cs

```diff
@@ -2427,6 +2427,7 @@ public sealed partial class DataGridCellItemAutomationPeer : System.Windows.Auto
         protected override bool HasKeyboardFocusCore() { throw null; }
         protected override bool IsContentElementCore() { throw null; }
         protected override bool IsControlElementCore() { throw null; }
+        protected override bool IsDialogCore() { throw null; }
```

Comment:
There should be a formal spec document, containing (among other things) the API additions mentioned in the reference files.   Do you have one?  If not, I've been doing a draft...

Reply:
I don't have one.

Reply:
See #4824